cookbooks/arcgis-enterprise/attributes/default.rb

```diff
@@ -25,12 +25,21 @@
   default['arcgis']['run_as_password'] = ENV['ARCGIS_RUN_AS_PASSWORD']
 end
 
-default['arcgis']['version'] = '10.7.1'
+default['arcgis']['run_as_msa'] = false
+
+default['arcgis']['version'] = '10.8'
 
 default['arcgis']['cache_authorization_files'] = false
 default['arcgis']['configure_windows_firewall'] = false
-if node['cloud']
+
+if node['cloud'] || ENV['arcgis_cloud_platform'] == 'aws'
```

Comment:
Should this line be changed to ` node['arcgis']['configure_cloud_settings']` like line 21 in attributes\datastore.rb

Reply:
@JoAlvarez To address https://github.com/Esri/arcgis-cookbook/issues/238 arcgis-enterprise cookbook v3.5.0 supports new arcgis/configure_cloud_settings attribute that determines if default values for other attributes are set in a cloud (AWS) specific way. 

By default arcgis/configure_cloud_settings attribute is set to true if `node['cloud'] || ENV['arcgis_cloud_platform'] == 'aws'`. Default values of some other attributes depend on  arcgis/configure_cloud_settings.